changes/389.fix.md

```diff
@@ -0,0 +1 @@
+Relax our direct dependnecy version range of aiohttp to allow the new 3.9 series for Python 3.12 support
```

Comment:
FYI the notes are supposed to be in RST, because towncrier puts them into `CHANGES.rst`.

Reply:
It's also a good idea to use the past tense, with the end-users being the target audience in mind.